components/list/Lists.vue

```diff
@@ -30,7 +30,7 @@ async function edit(listId: string) {
 </script>
 
 <template>
-  <CommonPaginator :end-message="false" :paginator="paginator">
+  <CommonPaginator end-message :paginator="paginator">
```

Comment:
```suggestion
  <CommonPaginator :paginator="paginator">
```

It seems that `end-message` defaults to `true`. Omitting this property still has the same effect but can be simpler.

https://github.com/elk-zone/elk/blob/59dda09cd479b14d8ccff843296b048b651784f3/components/common/CommonPaginator.vue#L15

Reply:
Thanks for the update!